recipes/zbarlight/__init__.py

```diff
@@ -0,0 +1,69 @@
+import os
+from toolchain import Recipe,shprint
+from os.path import join
+import sh
+import fnmatch
+from distutils.dir_util import copy_tree
+
+class ZbarLightRecipe(Recipe):
+    version = '1.2'
+    url = 'https://github.com/Polyconseil/zbarlight/archive/{version}.tar.gz'
+    library = "zbarlight.a"
+    depends = ['hostpython','python', 'libzbar']
+    pbx_libraries = ["libz","libbz2",'libc++','libsqlite3','CoreMotion']
+    include_per_arch = True
+
+    def get_zbar_env(self, arch):
+        build_env = arch.get_env()
+        dest_dir = join(self.ctx.dist_dir, "root", "python")
+        build_env["IOSROOT"] = self.ctx.root_dir
+        build_env["IOSSDKROOT"] = arch.sysroot
+        build_env["LDSHARED"] = join(self.ctx.root_dir, "tools", "liblink")
+        build_env["ARM_LD"] = build_env["LD"]
+        build_env["ARCH"] = arch.arch
+        build_env["C_INCLUDE_PATH"] = join(arch.sysroot, "usr", "include")
+        build_env["LIBRARY_PATH"] = join(arch.sysroot, "usr", "lib")
+        build_env['PYTHONPATH'] = join(dest_dir, 'lib', 'python2.7', 'site-packages')
+        build_env["CFLAGS"] = " ".join([
+            " -I{}".format(join(self.ctx.dist_dir, "include", arch.arch, "libzbar",'zbar')) +
+            " -arch {}".format(arch.arch)
+            ])
+        build_env['LDFLAGS'] += " -lios -lpython -lzbar"
+        return build_env
+
+    def build_arch(self, arch):
+        build_env = self.get_zbar_env(arch)
+        hostpython = sh.Command(self.ctx.hostpython)
+        shprint(hostpython, "setup.py", "build",
+                _env=build_env)
+        self.apply_patch("zbarlight_1_2.patch")#Issue getting the version, hard coding for now
+        self.biglink()
+
+    def install(self):
+        arch = list(self.filtered_archs)[0]
+        build_dir = join(self.get_build_dir(arch.arch),'build','lib.macosx-10.13-x86_64-2.7','zbarlight')
+        dist_dir  = join(self.ctx.dist_dir,'root','python','lib','python2.7','site-packages','zbarlight')
+        #Patch before Copying
+        #self.apply_patch("zbarlight_1_2.patch")#Issue getting the version, hard coding for now
```

Comment:
Minor: why is this patch line commented rather than being removed? The patch is already applied in `build_arch()`

Reply:
Oh I was trying to patch it after the file was moved.  It was easier to just patch the file before it was moved and I must have forgot to remove the line once I got the patch working before the move

Reply:
yep makes sense. You don't need to close the pull request if it's a little change. Simply update the pull request.
```sh
vim recipes/zbarlight/__init__.py
git add recipes/zbarlight/__init__.py
git commit --amend
git push
```